File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int

h = -1  # assigns h to an initial value that starts the loop
while (1 > h or h > 8):  # constantly prompts the user for an integer height between 1 and 8 inclusive
    h = get_int("Height: ")

for i in range(h):  # prints the pyramid
    print(" " * (h - i - 1), end="")
    print("#" * (i + 1), end="  ")
    print("#" * (i + 1))

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    int m;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        m = n - i ;
        for (int k = 1; k < m; k++)
        {
            printf(" ");
        }
        printf("#");
        for (int j = 0; j < i; j++)
        {
         printf("#");
        }
        printf("\n");
    }
}